### PR TITLE
Feat/chronograf: Add Chronograf

### PR DESCRIPTION
**Pull request by w-le**
This adds the Chronograf docker container, v1.9.

Chronograf is unfortunately required because:
- InfluxDBv2 frontend currently does [not support](https://github.com/influxdata/influxdb/issues/15721) a custom base href (e.g. [/influxdb](https://github.com/PlaceOS/nginx/blob/main/config/nginx.conf#L111)). When I tested influxdbv2, it worked fine at / with it's own subdomain, but I could not get it to work under any subdirectory - essential API calls would be made to an incorrect path.
- InfluxDBv2 OSS does not support OAuth, so we won't be able to use it's frontend for production deployments (super sad)

This PR:
- adds the Chrongraf container
- adds env vars that configure the local PlaceOS instance as an OAuth2 provider (but still requies 1 env var `GENERIC_CLIENT_ID` to be manually edited)
- Changes the influxDB image version to 2.0 alpine (so will use latest 2.0.x as opposed to 2.0.8 specifically)
- Maps the local influx volume to the new data path used by the dockerhub influx image (as opposed to the path used by the quay.io image).
- REQUIRES these new features in init: https://github.com/PlaceOS/init/pull/47 in order for installation to succeed
- REQUIRES the nginx.conf upstream change in: https://github.com/PlaceOS/nginx/pull/11 in order for the end user to access chronograf frontend

Note that the CI builds will be expected to fail until the above linked PRs in init and nginx are merged, released and utilised by this branch of partner-environment.

### CODE REVIEW

**Comment by w-le** [comment]
current status is that I'm trying to test e2e locally but `docker pull ghcr.io/placeos/init:featchronograf` is timing out (along with all  `docker pull ghcr.io...`. I think it's ghcr, will try again tomorrow.

**Review by stakach — APPROVED** [review]
LGTM

**Comment by w-le** [comment]
oh @stakach  but this changes REQUIRES the 2 upstream linked PRs in my OP (last two bullet points).

So either:
- those 2 PRs get reviewed and merged very soon
- OR, we revert this merge, and wait until those 2 PRs are merged first

**Comment by caspiano** [comment]
It's going to remain broken until we cut the next release. Perhaps a revert is in order and then we can ensure backwards compatibility is preserved

**Comment by w-le** [comment]
yep agree, revert

**Pull request merged**
